test/e2e/lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -245,25 +245,15 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 		const trashLocator = By.css( 'button.editor-post-trash' );
 
 		await this.selectDocumentTab();
-		await driverHelper.waitUntilElementLocatedAndVisible( this.driver, trashLocator );
 		await driverHelper.clickWhenClickable( this.driver, trashLocator );
 
-		// wait for 'Move to trash' button to disappear
-		try {
-			return await driverHelper.waitUntilElementNotLocated(
-				this.driver,
-				trashLocator,
-				this.explicitWaitMS * 3
-			);
-		} catch ( e ) {
-			// if it's still present send slack notification, post is deleted
-			return await SlackNotifier.warn(
-				'"Move to trash" button is still present, but post is deleted.',
-				{
-					suppressDuplicateMessages: true,
-				}
-			);
-		}
```

Comment:
This whole part is outdated. It wouldn't fail the test, but it would send a false warning about the post deletion. Instead of that, we just need to switch the context because we're leaving Gutenberg and make sure we've landed on the Posts page (see below).